tryit.py:21: error: Unsupported operand types for < ("NaiveDateTime" and "AwareDateTime")  [operator]
tryit.py:21: note: Following member(s) of "AwareDateTime" have conflicts:
tryit.py:21: note:     Expected:
tryit.py:21: note:         def timetz(self) -> Time[None]
tryit.py:21: note:     Got:
tryit.py:21: note:         def timetz(self) -> Time[tzinfo]
tryit.py:21: note:     tzinfo: expected "None", got "tzinfo"
tryit.py:23: error: Incompatible types in assignment (expression has type "AwareDateTime", variable has type "NaiveDateTime")  [assignment]
tryit.py:23: note: Following member(s) of "AwareDateTime" have conflicts:
tryit.py:23: note:     Expected:
tryit.py:23: note:         def timetz(self) -> Time[None]
tryit.py:23: note:     Got:
tryit.py:23: note:         def timetz(self) -> Time[tzinfo]
tryit.py:23: note:     tzinfo: expected "None", got "tzinfo"
tryit.py:26: error: Incompatible types in assignment (expression has type "NaiveDateTime", variable has type "AwareDateTime")  [assignment]
tryit.py:26: note: Following member(s) of "NaiveDateTime" have conflicts:
tryit.py:26: note:     Expected:
tryit.py:26: note:         def timetz(self) -> Time[tzinfo]
tryit.py:26: note:     Got:
tryit.py:26: note:         def timetz(self) -> Time[None]
tryit.py:26: note:     tzinfo: expected "tzinfo", got "None"
tryit.py:34: error: Incompatible types in assignment (expression has type "NaiveDateTime", variable has type "Date")  [assignment]
tryit.py:35: error: Incompatible types in assignment (expression has type "AwareDateTime", variable has type "Date")  [assignment]
tryit.py:45: error: Unsupported operand types for < ("NaiveDateTime" and "DateTime[tzinfo]")  [operator]
tryit.py:45: note: Following member(s) of "DateTime[tzinfo]" have conflicts:
tryit.py:45: note:     Expected:
tryit.py:45: note:         def timetz(self) -> Time[None]
tryit.py:45: note:     Got:
tryit.py:45: note:         def timetz(self) -> Time[tzinfo]
tryit.py:45: note:     tzinfo: expected "None", got "tzinfo"
tryit.py:46: error: Unsupported operand types for < ("AwareDateTime" and "DateTime[None]")  [operator]
tryit.py:46: note: Following member(s) of "DateTime[None]" have conflicts:
tryit.py:46: note:     Expected:
tryit.py:46: note:         def timetz(self) -> Time[tzinfo]
tryit.py:46: note:     Got:
tryit.py:46: note:         def timetz(self) -> Time[None]
tryit.py:46: note:     tzinfo: expected "tzinfo", got "None"
tryit.py:48: error: "tzinfo" has no attribute "key"  [attr-defined]
tryit.py:53: error: Incompatible types in assignment (expression has type "DateTime[timezone]", variable has type "NaiveDateTime")  [assignment]
tryit.py:53: note: Following member(s) of "DateTime[timezone]" have conflicts:
tryit.py:53: note:     Expected:
tryit.py:53: note:         def timetz(self) -> Time[None]
tryit.py:53: note:     Got:
tryit.py:53: note:         def timetz(self) -> Time[timezone]
tryit.py:53: note:     tzinfo: expected "None", got "timezone"
tryit.py:59: error: No overload variant of "__add__" of "DateTime" matches argument type "NaiveDateTime"  [operator]
tryit.py:59: note: Possible overload variants:
tryit.py:59: note:     def __add__(self, timedelta, /) -> NaiveDateTime
tryit.py:59: note:     def __add__(self, relativedelta, /) -> NaiveDateTime
Found 10 errors in 1 file (checked 1 source file)
